security/compliance_operator/compliance-operator-crd.adoc

```diff
@@ -0,0 +1,509 @@
+    compliance.openshift.io/check-status: FAIL
+    compliance.openshift.io/suite: example-compliancesuite
+    compliance.openshift.io/scan-name: workers-scan
+  name: workers-scan-no-direct-root-logins
+  namespace: openshift-compliance
+  ownerReferences:
+  - apiVersion: compliance.openshift.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: ComplianceScan
+    name: workers-scan
+description: <description of scan check>
+instructions: <manual instructions for the scan>
+id: xccdf_org.ssgproject.content_rule_no_direct_root_logins
+severity: medium <1>
+status: FAIL <2>
+----
+
+<1> Describes the severity of the scan check.
+<2> Describes the result of the check. The possible values are:
+* PASS: indicates the check was successful.
+* FAIL: indicates the check was unsuccessful.
+* INFO: indicates the check was successful and found something not severe enough to be considered an error.
+* MANUAL: indicates the check cannot automatically assess the status and manual check is required.
+* INCONSISTENT: indicates that different nodes report different results.
+* ERROR: indicates the check run successfully, but could not complete.
+* NOTAPPLICABLE: indicates the check did not run as it is not applicable.
+
+To get all the check results from a suite, run the following command:
+[source,terminal]
+----
+oc get compliancecheckresults -l compliance.openshift.io/suite=<suit name>
+----
+
+[id="compliance-remediation-object_{context}"]
+=== ComplianceRemediation object
+For a specific check you can have a datastream specified fix. However, if a Kubernetes fix is available, then the Compliance Operator creates a `ComplianceRemediation` object.
+
+.Example `ComplianceRemediation` object
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceRemediation
+metadata:
+  labels:
+    compliance.openshift.io/suite: example-compliancesuite
+    compliance.openshift.io/scan-name: workers-scan
+    machineconfiguration.openshift.io/role: worker
+  name: workers-scan-disable-users-coredumps
+  namespace: openshift-compliance
+  ownerReferences:
+  - apiVersion: compliance.openshift.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: ComplianceCheckResult
+    name: workers-scan-disable-users-coredumps
+    uid: <UID>
+spec:
+  apply: false <1>
+  object:
+    current: <2>
+       apiVersion: machineconfiguration.openshift.io/v1
+       kind: MachineConfig
+       spec:
+         config:
+           ignition:
+             version: 2.2.0
+           storage:
+             files:
+             - contents:
+                 source: data:,%2A%20%20%20%20%20hard%20%20%20core%20%20%20%200
+               filesystem: root
+               mode: 420
+               path: /etc/security/limits.d/75-disable_users_coredumps.conf
+    outdated: {} <3>
+----
+
+<1> `true` indicates the remediation was applied. `false` indicates the remediation was not applied.
```

Comment:
"Solution" rather than "remediation" for the rest of the topic?

Reply:
@lpettyjo remediations is the name of the CR.